wordpress-mobile/WordPress-iOS
# New stories intro assets and links

Adds new assets and links for the Stories intro screen.

The iOS counterpart of https://github.com/wordpress-mobile/WordPress-Android/pull/13526

### Testing

* Freshly install the WordPress app (or delete the `storiesIntroWasAcknowledged` User Default).
* Tap the FAB from your Site Details screen.
* Select the "Story post" post option.
* Ensure that tapping the demo stories (shown below) opens and loads the demo stories.
* Ensure the images below are shown.

![Screen Shot 2021-02-07 at 9 56 42 PM](https://user-images.githubusercontent.com/3250/107177596-6de69580-698f-11eb-8ed8-e0137a8db29d.png)

PR submission checklist:
